.github/workflows/ci.yml

```diff
@@ -72,3 +72,22 @@ jobs:
           which go
           go version
           go test -race -timeout 60s ./...
+  lint:
```

Comment:
Please revert changes to `ci.yml`. Linting tasks are included in a separate config.

Reply:
Sure, but I added here because the linter is executed only on push, not on PRs, should I enable on_pull_request in them?

Reply:
👍 Sure, you can add that to the `linter.yml`.

Reply:
done :)